source/NVDAObjects/IAccessible/winword.py

```diff
@@ -212,6 +214,16 @@ def fetchAssociatedHeaderCellText(self,cell,columnHeader=False):
 			if text:
 				return text
 
+	@script(
+		gesture="kb:NVDA+shift+c",
+		# Translators: The label of a shortcut of NVDA.
+		description=_(

```

Comment:
```suggestion
		description=_(
			"Set column header"
```

Reply:
I only copied the description from the old syntax, but I accepted your suggestion thinking that it is indeed better to shorten it.

Reply:
Hi, my intention for this suggestion was to add a shorter description before the longer description, not to remove it entirely.
If you want to remove this longer description please ensure there it is included in the user guide.

Reply:
ah I see it is